### PR TITLE
Refresh desktop/developers

### PR DESCRIPTION
**Pull request by petesfrench**
**Note: There are separate PRs for the changes to the [D3 graphs ](https://warthogs.atlassian.net/browse/WD-8270)and for the logic needed to allow pagination in the [latest-news module](https://warthogs.atlassian.net/browse/WD-8426)**
## Done

- Applies the [content](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit) and [design](https://www.figma.com/file/wLOtPeQhAiExhQCwKYhi3H/24.04-U.com---desktop-(rebranding)?type=design&node-id=1-2&mode=design&t=Fb5N2muecteluj8U-0) changes

## QA

- Go to https://ubuntu-com-13491.demos.haus/desktop/developers
- Compare against design and copydoc
- Check on different sized screens

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7733

### CODE REVIEW

**Comment by petesfrench** [comment]
Hey @lyubomir-popov, wanted to get up what I have so far as I need advice regarding the logo sections. I was able to use a `--dense` version of [MPTs icon section](https://github.com/canonical/ubuntu.com/blob/desktop-refresh/static/sass/styles.scss#L1405) and I think it is acceptabe. But for the other sections, I tried to size the logo sections appropriately and had some trouble, ie:
image
And I didn't try with 'Access to cutting-edge hardware and software' as I wanted input first.
Can you help me out with these icons?

File: templates/desktop/developers.html [hunk 1]
```diff
@@ -1,658 +1,644 @@
...
+      articleTemplateSelector: "#desktop-articles-template",
+      excerptLength: 200,
+      groupId: "1479",
+      linkImage: true,
+      limit: 3
+    }
+    )
+</script>
+
+{% include "shared/contextual_footers/_desktop.html" %}
```
Comment (mtruj013) [inline templates/desktop/developers.html:641]:
```suggestion

```
I believe this one is only shared by the landing page and /features

**Review by mtruj013 — APPROVED** [review]
Thanks @petesfrench! Few comments, all for pretty minor stuff. About the hrs, this cycle I've gotten a lot of feedback that we should be closing all elements [per convention](https://webteam.canonical.com/practices/html-code-standards) so that's why I flagged

**Comment by petesfrench** [comment]
> Thanks @petesfrench! Few comments, all for pretty minor stuff. About the hrs, this cycle I've gotten a lot of feedback that we should be closing all elements [per convention](https://webteam.canonical.com/practices/html-code-standards) so that's why I flagged

I had been wondering about this recently, good to know it is documented. I have added it as a topic in our dev-discussion meeting, as this is something that can easily be enforced/automated.

**Comment by juanruitina** [comment]
As discussed, let's drop the pagination on the news, it can remain this way.

Some visual bits I've noticed:

- [ ] Ubuntu Pro logo should be smaller, it's 42px high on the design.
- [ ] The hero image should be cropped as per the Figma file. I exported the image [here](https://drive.google.com/drive/folders/1S1zhEhivKDc0f4Xwl8JjEUFIwVGzftuy?usp=sharing).
- [ ] We should use variable width logos. I exported all of them [here](https://drive.google.com/drive/folders/1S1zhEhivKDc0f4Xwl8JjEUFIwVGzftuy?usp=sharing), but some might have been used elsewhere, so perhaps already in Assets.
- [ ] Please add some spacing between icons when they wrap, `row-gap: 1rem` on the flex container should do the trick.

**Pull request merged**